src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -183,12 +190,22 @@ const IssueForm = (): JSX.Element | null => {
     }
   }, [selectVaultManually, vault, setError, clearErrors, t, btcAmount, feeRate]);
 
-  if (status === STATUSES.IDLE || status === STATUSES.PENDING || requestLimitsIdle || requestLimitsLoading) {
+  if (
+    status === STATUSES.IDLE ||
+    status === STATUSES.PENDING ||
+    requestLimitsIdle ||
+    requestLimitsLoading ||
+    governanceTokenBalanceIdle ||
+    governanceTokenBalanceLoading
+  ) {
     return <PrimaryColorEllipsisLoader />;
   }
   if (requestLimits === undefined) {
     throw new Error('Something went wrong!');
   }
+  if (governanceTokenBalance === undefined) {
```

Comment:
Cant this be included in the line 203 check?

Reply:
Done!